DSCResources/MSFT_xNetworkTeam/MSFT_xNetworkTeam.psm1

```diff
@@ -0,0 +1,230 @@
+﻿if (Test-Path "${PSScriptRoot}\${PSUICulture}")
+{
+    Import-LocalizedData -BindingVariable LocalizedData -filename MSFT_xNetworkTeam.psd1 -BaseDirectory "${PSScriptRoot}\${PSUICulture}"
+}
```

Comment:
@rchaganti 
No, I'm fine with base directory.  It is ok to give the cmdlet a hint on where to look.  But, it seem we should leave the language search up to the cmdlet (unless we have a really good reason,) as it should be well used and tested code.

FYI, you have a merge conflict